model/publishing/delivery/tasks/DeployTestEnvironments.php

```diff
@@ -173,4 +171,55 @@ private function getLoggerService(): LoggerService
     {
         return $this->getServiceLocator()->get(LoggerService::SERVICE_ID);
     }
+
+    /**
+     * @param core_kernel_classes_Resource $delivery
+     * @return array
+     * @throws \core_kernel_persistence_Exception
+     */
+    private function prepareRequestData(core_kernel_classes_Resource $delivery): array
+    {
+        $qtiPackageFile = $this->getQtiTestPackageFile($delivery);
+        $streamData = [
+            [
+                'name' => RestTest::REST_FILE_NAME,
+                'contents' => $qtiPackageFile->readStream(),
+            ],
+            [
+                'name' => RestTest::REST_IMPORTER_ID,
+                'contents' => 'taoQtiTest',
+            ],
+            [
+                'name' => RestTest::REST_DELIVERY_PARAMS,
+                'contents' => json_encode(
+                    [
+                        PublishingDeliveryService::ORIGIN_DELIVERY_ID_FIELD => $delivery->getUri()
```

Comment:
Should we include `PublishingDeliveryService::ORIGIN_TEST_ID_FIELD` 
would be inserted on RDF of remote delivery and can be useful in case of debugging.

Reply:
AFAIK we don't have handler for this parameter in receiving API. It can be added as improvement separately.